app/models/player_stat.rb

```diff
@@ -4,4 +4,19 @@ class PlayerStat < ApplicationRecord
   def stat_dates
     self.created_at.strftime('%b-%d, %Y')
   end
+
+  def self.player_stats(player, x = 10)
+    x = create_limit(x)
+    PlayerStat.where(player_profile_id: player.profile.id)
+    .order(created_at: :DESC)
+    .limit(x)
+  end
+
+  def self.create_limit(x)
+    if x == nil
```

Comment:
@thedanielvogelsang 

Looks good so far, what do you think about refactoring self.create_limit to :
x = 10 if x.nil?

Reply:
I like that more. If we can confirm this merge, I'll refactor locally for the next branch PR